src/lib/state/actions/communities.js

```diff
@@ -0,0 +1,52 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2022 CERN.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import axios from 'axios';
+import {
+  FETCH_COMMUNITY_ERROR,
+  FETCH_COMMUNITY_LOADING,
+  FETCH_COMMUNITY_SUCCESS,
+  SET_COMMUNITY,
+} from '../types';
+
+const apiConfig = {
+  withCredentials: true,
+  xsrfCookieName: 'csrftoken',
+  xsrfHeaderName: 'X-CSRFToken',
+};
+const axiosWithconfig = axios.create(apiConfig);
+
+export const getCommunity = (communityId) => {
+  return async (dispatch, getState, config) => {
+    dispatch({
+      type: FETCH_COMMUNITY_LOADING,
+    });
+    try {
+      const response = await axiosWithconfig.get(
+        `/api/communities/${communityId}`
```

Comment:
hardcoded url

Reply:
Not sure, what's the best practice to avoid it ... any suggestions?

Reply:
No need anymore